low-level/message_handlers/service_msg_handler.py

```diff
@@ -141,6 +160,21 @@ def _process_msg(self, jsonMsg):
                 .get("service_controller").get("service_request")
             request = f"{service_request}:{service_name}"
 
+            error_info = {}
+            if service_name not in self.monitored_services:
+                logger.error(f"{service_name} - service is not present in monitored_services list," +\

```

Comment:
`+\` 
We can avoid `\` with `+` operator.  String can be concatenated even `\` is not mentioned. Applicable wherever.

Reply:
Done.